tests/integration/test_ols.py

```diff
@@ -101,3 +105,53 @@ def test_post_question_on_unknown_response_type() -> None:
         assert response.json() == {
             "detail": {"response": "Internal server error. Please try again."}
         }
+
+
+class TestQuery:
+    """Test the /v1/query endpoint."""
+
+    def test_unsupported_provider_in_post(self):
+        """Check the REST API /v1/query with POST method when unsupported provider is requested."""
+        config.llm_config.providers = {}  # empty config - no providers
```

Comment:
is this mutating the config for other tests that run later?

ideally we want each test to re-initialize the config w/ whatever it needs to ensure there aren't inadvertent side effects